graphql/types/OrderForm.graphql

```diff
@@ -22,6 +23,20 @@ type OrderForm {
   pickupPointCheckedIn: PickupPoint
 }
 
+type CustomData {
+  customApps: [CustomApps]
+}
+
+type CustomApps {
+  fields: Fields
+  id: String
+  major: Int
+}
+
+type Fields {
+  substituteType: String
```

Comment:
Since this is a custom field. It should be a key value pair instead of a fixed one.